source/templates/workflows.slim

```diff
@@ -182,4 +190,50 @@
+			header
+				h2 == data[:strings][:workflows][:inputs][:insert_variable][:action]
+				input[type="text" ng-model="insertVariableCtrl.keyFilter" placeholder="#{ data[:strings][:workflows][:inputs][:insert_variable][:key_filter_placeholder] }"]
+			article.vertical-scroll
+				p.progress-indicator[progress-model="insertVariableCtrl.loadInsertableVariablesProgress"]
+				div[ng-if="insertVariableCtrl.loadInsertableVariablesProgress.isIdle"]
+					ul.variable-source[ng-repeat="variablesOfSource in insertVariableCtrl.insertableVariables"]
+						li[ng-repeat="variable in variablesOfSource.variables | variablesfilteredByKey: insertVariableCtrl.keyFilter" ng-if="insertVariableCtrl.shouldShowVariable(variable)"]
+							button[ng-click="insertVariableCtrl.variableSelected(variable)"]
+								strong[ng-bind="variable | prettifiedVariableKey"]
+								em[ng-bind="variablesOfSource.title"]
+	popup[model="workflowsCtrl.insertSecretToStepInputConfig.popup"]
+		.popup-body.snowflake#insert-variable-popup-body[ng-if="workflowsCtrl.insertSecretToStepInputConfig.popup.isVisible" ]
+			header
+				h2 == data[:strings][:workflows][:inputs][:insert_secret][:action]
+			article.vertical-scroll
+				.create-new-secret[ng-controller="CreateAndInsertSecretController as createAndInsertSecretCtrl" ng-init="createAndInsertSecretCtrl.init(workflowsCtrl.insertSecretToStepInputConfig.secretSelected, workflowsCtrl.insertSecretToStepInputConfig.workflows, workflowsCtrl.insertSecretToStepInputConfig.selectedWorkflow, workflowsCtrl.insertSecretToStepInputConfig.selectedStep)"]
+					.title == data[:strings][:workflows][:inputs][:insert_secret][:create_new_secret_env_var_label]
+					form[name="newSecretEnvVarForm" ng-submit="createAndInsertSecretCtrl.saveAndInsertSecretEnvVar()" ng-if="!createAndInsertSecretCtrl.newSecretEnvVar.customProperties.progress.isInProgress" autocomplete="off" autocapitalize="none"]
+						.key-value-delete
+							input.key[type="text" name="key" ng-model="createAndInsertSecretCtrl.newSecretEnvVar.key" ng-model-options="{getterSetter: true}" ng-disabled="!createAndInsertSecretCtrl.newSecretEnvVar.isKeyChangeable" custom-validators="{'key': createAndInsertSecretCtrl.isValidKey(createAndInsertSecretCtrl.newSecretEnvVar.key(), appService.secrets)}" placeholder="#{ data[:strings][:env_vars][:key_placeholder] }" autocomplete="off" autocorrect="off" autocapitalize="none" spellcheck="false" skip-drag-order]
+							.value-with-actions.secret
+								textarea.value[name="value" msd-elastic ng-model="createAndInsertSecretCtrl.newSecretEnvVar.value" ng-model-options="{getterSetter: true}" ng-class="{'protected' : requestService.mode == 'website' && createAndInsertSecretCtrl.newSecretEnvVar.isProtected()}" ng-disabled="!createAndInsertSecretCtrl.newSecretEnvVar.shouldShowValue" ng-trim="false" placeholder="#{ data[:strings][:env_vars][:value_placeholder] }" skip-drag-order]
+						.env-var-settings
```

Comment:
Checkboxes not working, always setting true for "Replace variables in inputs?" and false for "Expose for Pull Requests?".